<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-08-15T12:59:08 EET" name="Suite" finished-at="2022-08-15T13:01:28 EET" duration-ms="139987">
    <groups>
    </groups>
    <test started-at="2022-08-15T12:59:08 EET" name="Test" finished-at="2022-08-15T13:01:28 EET" duration-ms="139987">
      <class name="tests.FlowardTest">
        <test-method is-config="true" signature="startReport()[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T12:59:08 EET" name="startReport" finished-at="2022-08-15T12:59:08 EET" duration-ms="316" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startReport -->
        <test-method is-config="true" signature="setup(java.lang.reflect.Method)[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T12:59:08 EET" name="setup" finished-at="2022-08-15T12:59:15 EET" duration-ms="7339" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.FlowardTest.AddToCartTest() throws java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="init()[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T12:59:15 EET" name="init" finished-at="2022-08-15T12:59:15 EET" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="AddToCartTest()[pri:1, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T12:59:15 EET" name="AddToCartTest" description="Add to cart" finished-at="2022-08-15T13:00:16 EET" duration-ms="60464" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddToCartTest -->
        <test-method is-config="true" signature="setTestResult(org.testng.ITestResult)[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:00:16 EET" name="setTestResult" finished-at="2022-08-15T13:00:16 EET" duration-ms="3" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=AddToCartTest status=SUCCESS method=FlowardTest.AddToCartTest()[pri:1, instance:tests.FlowardTest@a1153bc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setTestResult -->
        <test-method is-config="true" signature="setup(java.lang.reflect.Method)[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:00:16 EET" name="setup" finished-at="2022-08-15T13:00:26 EET" duration-ms="10588" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void tests.FlowardTest.AddToCartTestFail() throws java.lang.InterruptedException,java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="init()[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:00:26 EET" name="init" finished-at="2022-08-15T13:00:26 EET" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="AddToCartTestFail()[pri:1, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:00:26 EET" name="AddToCartTestFail" description="Add to cart" finished-at="2022-08-15T13:01:27 EET" duration-ms="60434" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[item didn't add to cart expected [This is wrong text] but found [Blossom Farm Activity Playgym]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: item didn't add to cart expected [This is wrong text] but found [Blossom Farm Activity Playgym]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at tests.FlowardTest.AddToCartTestFail(FlowardTest.java:81)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddToCartTestFail -->
        <test-method is-config="true" signature="setTestResult(org.testng.ITestResult)[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:01:27 EET" name="setTestResult" finished-at="2022-08-15T13:01:27 EET" duration-ms="260" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=AddToCartTestFail status=FAILURE method=FlowardTest.AddToCartTestFail()[pri:1, instance:tests.FlowardTest@a1153bc] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setTestResult -->
        <test-method is-config="true" signature="close()[pri:0, instance:tests.FlowardTest@a1153bc]" started-at="2022-08-15T13:01:27 EET" name="close" finished-at="2022-08-15T13:01:28 EET" duration-ms="516" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- tests.FlowardTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
